frontend/src/pages/VacationPlanner.tsx

```diff
@@ -211,9 +248,9 @@ export const VacationPlanner = () => {
         setToastList([
           ...toastList,
           {
-            type: "info",
+            type: "warning",
             timeout: 10000,
-            message: "Vacation has already been reported on this period.",
+            message: "Time entries have already been reported on this period.",
```

Comment:
I think, I'd go for "vacation" here, it's more how the user would think about it. "Time entries" is probably more correct, but to me it sounds like the technical term we developers use rather than what the user has in mind.

Reply:
Vacation is not accurate since we check for any type of entry. I rephrased to "time", so that is more generic